docs/docusaurus.config.ts

```diff
@@ -173,11 +177,14 @@ const config: Config = {
     [
       '@docusaurus/preset-classic',
       {
+        // Releases to `docs.dagster.io` are triggered on pushes to branches prefixed with
+        // `release-*`, therefore the most recent release should be labeled as the version in that
+        // branch (eg. `release-1.10.5`). Ultimately we will automate this process.
         docs: {
           lastVersion: 'current',
           versions: {
             current: {
-              label: '1.10.5 (master)',
+              label: '1.10.5',
```

Comment:
I think this will be correct as of this week, so you may want until the end of the week to merge